locales/fake/messages.json

```diff
@@ -0,0 +1,6 @@
+{
+    "language_default_pretty": {
```

Comment:
Or to be more precise: What is this used for?

Reply:
In English language_default_pretty is fully defined: "System Default ($t(detectedLanguage))"

This works as a fallback as many languages have language_default translated.

messing with fallback might be a way to rename a key, and not lose the translated content. 

tabLanding
defaultChangelogAction
defaultPrivacyPolicyAction
tabHelp
tabFirmwareFlasher

Reply:
Not sure I understand this right, but the way `fallbackLng` is used it looks now like `fake` is the only language that will ever be used as fallback language. This is probably not what we want.

Also, should this mythical language maybe be called `fallback` or similar instead of `fake`?

Reply:
I believe the fallback is to english. We can specify multiple fallback languages. So for `defaultDonateBottom` on the landing page gets english when using german. (my change to the link ended up wiping out the translations).

Reply:
Have a look at the code. Only the first element of `fallbackLng` seems to ever be used. Is it possible that English is a hardcoded default fallback for the framework we are using?

Reply:
In i18next or in our code? We **are** injecting information (resources) into the first fallbackLng, but I figured that would be okay to ignore. I figure it'd be an annoying be problem to throw out a language while running, even if its a fake one.

For this screenshot I made did `fallbackLng: ['fallback','de','en'],` and then loaded polish translation manually `i18next.changeLanguage('pl')`
![image](https://user-images.githubusercontent.com/190571/62745448-d6059580-b9ff-11e9-9a33-185fdeeeeaf8.png)

Reply:
Sorry, my bad, I'd assumed `fallbackLng` was an option that we'd added (dodgy name and all).